src/rf/apps/core/models.py

```diff
@@ -133,6 +135,30 @@ class Meta:
     thumb_large_key = CharField(max_length=255, blank=True, default='',
                                 help_text='S3 key for large thumbnail')
 
+    def retry_possible(self):
+        """
+        Returns true if it is possible to retry processing a layer.
+        """
+        return self.status == enums.STATUS_FAILED
```

Comment:
Do you think retry should be available in the case of **all** failures? If the image is not a valid tif or if it doesn't actually have three bands should we bother doing all the work again? It seems to be as though we should only retry if we got an unexpected error during validation or if we failed somewhere in the EMR part of the job.